src/app/(site)/login/page.tsx

```diff
@@ -1,59 +1,89 @@
-"use client";
+'use client';
 
-import { FC, useState } from "react";
-import Link from "next/link";
+import { FC, useState } from 'react';
+import Link from 'next/link';
+import { useAuth } from '@/app/context/AuthContext';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+	faPersonCircleExclamation,
+	faSpinner,
+} from '@fortawesome/free-solid-svg-icons';
 
 const LoginForm: FC = () => {
-  const [userData, setUserData] = useState({
-    email: "",
-    password: "",
-  });
-  console.log(userData);
```

Comment:
delete console.log?

Reply:
I deleted it because it's not needed and it's properly connected to the AuthContext